frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/index.tsx

```diff
@@ -149,49 +157,61 @@ const DatasetRow: FC<Props> = ({
     <StyledRow>
       <DetailsCell>
         <TitleContainer>
-          <StyledRadio
-            onChange={() => onSelect(dataset.id)}
-            checked={selected === dataset.id}
-          />
-          <div>{name}</div>
+          <span>{name}</span>
           {!isLoading && (
             <ErrorTooltip isFailed={isFailed} error={error} type={type} />
           )}
+          {isLoading && (
+            <AsyncUploadStatus
```

Comment:
this is just removing `datasetId` and `collectionId` from `< AsyncUploadStatus />`, since they were used for canceling an uploading dataset next to the loader, which is removed

<img width="453" alt="Screen Shot 2021-03-31 at 12 17 08 PM" src="https://user-images.githubusercontent.com/6309723/113198968-6ae38500-921b-11eb-9573-198338b04642.png">